network/topics/scoring.go

```diff
@@ -33,16 +33,20 @@ func scoreInspector(logger *zap.Logger, scoreIdx peers.ScoreIndex) pubsub.Extend
 			filtered := make(map[string]*pubsub.TopicScoreSnapshot)
 			var totalInvalidMessages float64
 			var totalLowMeshDeliveries int
+			var totalMeshDeliveriesSquaredDeficitSum float64
+			meshDeliveryCounters := make([]float64, 0)
 			for topic, snapshot := range peerScores.Topics {
 				if snapshot.InvalidMessageDeliveries != 0 {
 					filtered[topic] = snapshot
 				}
 				if snapshot.InvalidMessageDeliveries > 0 {
 					totalInvalidMessages += math.Sqrt(snapshot.InvalidMessageDeliveries)
 				}
-				if snapshot.MeshMessageDeliveries < 107 {
+				if snapshot.MeshMessageDeliveries < 11.243655244810899 {
```

Comment:
what is the calculation for this non round number?
can we have fractional message deliveries?

Reply:
This is approximately the threshold. Just a log for testing purposes. Will be removed afterwards.
Yup, fractional due to decay.